src/views/Settings.vue

```diff
@@ -8,68 +8,168 @@
-      </ion-item>
+      <div class="user-profile">
+        <ion-card>
+          <ion-item lines="full">
+            <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
+              <Image :src="userProfile?.partyImageUrl"/>
+            </ion-avatar>
+            <ion-label>
+              {{ userProfile?.partyName }}
+              <p>{{ userProfile?.userLoginId }}</p>
+            </ion-label>
+          </ion-item>
+          <ion-button fill="outline" color="danger" @click="logout()">{{ $t("Logout") }}</ion-button>
+          <!-- Commenting this code as we currently do not have reset password functionality -->
+          <!-- <ion-button fill="outline" color="medium">{{ $t("Reset password") }}</ion-button> -->
+        </ion-card>
+      </div>
+      
+      <h1>{{ $t('OMS') }}</h1>
+
+      <section>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
+              {{ $t("OMS instance") }}
+            </ion-card-subtitle>
+            <ion-card-title>
+              {{ instanceUrl }}
+            </ion-card-title>
+          </ion-card-header>
+          
+          <ion-card-content>
+            {{ $t('This is the name of the OMS you are connected to right now. Make sure that you are connected to the right instance before proceeding.') }}
+          </ion-card-content> 
+           
+          <ion-button @click="goToOms" fill="clear">
+            {{ $t('Go to OMS') }}
+            <ion-icon slot="end" :icon="openOutline" />
+          </ion-button>
+        </ion-card>
+      </section>
+
+      <hr />
+
+      <h1>{{ $t('App') }}</h1>
+      
+      <section>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>
+              {{ $t('Timezone') }}
+            </ion-card-title>
+          </ion-card-header>
+          
+          <ion-card-content>
+            {{ $t('The timezone you select is used to ensure automations you schedule are always accurate to the time you select.') }}
+          </ion-card-content> 
+           
+          <ion-item lines="none">
+            <ion-label> {{ userProfile && userProfile.userTimeZone ? userProfile.userTimeZone : '-' }} </ion-label>
+            <ion-button @click="changeTimeZone()" slot="end" fill="outline" color="dark">{{ $t("Change") }}</ion-button>
+          </ion-item>
+        </ion-card>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
+              {{ $t("File upload")}}
+            </ion-card-subtitle>
+            <ion-card-title>
+              {{ $t("Date Format") }}
+            </ion-card-title>
+          </ion-card-header>
+          
+          <ion-card-content>
+            {{ $t('Enter a custom date time format that you want to use when uploading documents to HotWax Commerce.') }}
+            <p>{{ $t('Luxon date time formats can be found') }} <a href="https://github.com/moment/luxon/blob/master/docs/formatting.md">{{ $t("here") }}</a></p>
```

Comment:
Change luxon link to `https://moment.github.io/luxon/#/formatting?id=presets`

Reply:
Updated url